lms/djangoapps/discussion/static/discussion/templates/search.underscore

```diff
@@ -0,0 +1,9 @@
+<label class="field-label sr-only" for="search" id="search-hint">Search all posts</label>
```

Comment:
Can you update the UX reference page to match your newer HTML:

https://github.com/edx/edx-platform/blob/master/lms/templates/ux/reference/course-skeleton.html